maint/local_python/binding_c.py

```diff
@@ -583,6 +653,50 @@ def dump_copy_right():
     G.out.append(" */")
     G.out.append("")
 
+def dump_profiling_qmpi(func, mapping):
+    parameters = ""
+    for p in func['c_parameters']:
+        parameters = parameters + ", " + p['name']
+
+    func_name = get_function_name(func, mapping)
+    s = get_declare_function(func, mapping)
+
+    dump_line_with_break(s)
+    G.out.append("{")
+    G.out.append("    QMPI_Context context;")
+    G.out.append("    Q%s_t *fn_ptr;" % (func_name))
+    G.out.append("")
+    G.out.append("    context.storage_stack = NULL;")
+    G.out.append("")
+    if func_name == "MPI_Pcontrol":
+        G.out.append("    va_list varargs;")
+        G.out.append("    va_start(varargs, level);")
+        G.out.append("")
+    elif "initcheck" in func['skip']:
```

Comment:
Now that I think about it more, this probably should be an `if` and not an `elif`. I suppose `MPI_PCONTROL` can be called before MPI is initialized?